cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -41,6 +41,7 @@ resources:
 - ../../../../base/core/namespaces/kubeflow
 - ../../../../base/core/namespaces/manageiq
 - ../../../../base/core/namespaces/neu-cs6620
+- ../../../../base/core/namespaces/octo-et-security-demo-next
```

Comment:
Again in the issue thread you referenced the IONOS cluster which would be Jerry. Is this project going on Smaug or Jerry @durandom?

Reply:
its going to smaug